SCClassLibrary/Common/Collections/Dictionary.sc

```diff
@@ -560,12 +560,32 @@ IdentityDictionary : Dictionary {
 		^this.superPerformList(\doesNotUnderstand, selector, args);
 	}
 
-		// Quant support.
-		// The Quant class assumes the quant/phase/offset scheduling model.
-		// If you want a different model, you can write a dictionary like so:
-		// (nextTimeOnGrid: { |self, clock| ... calculate absolute beat number here ... },
-		//	parameter: value, parameter: value, etc.)
-		// If you leave out the nextTimeOnGrid function, fallback to quant/phase/offset.
+	doesNotUnderstandWithKeys {|selector, argsArray, keywordArgsAsPairs|
+		if(know.not){
+			^this.superPerformList(\doesNotUnderstandWithKeys, selector, argsArray, keywordArgsAsPairs )
+		};
+
+		this[selector] !? {|f|
+			^f.functionPerformWith(\value, [this] ++ argsArray, keywordArgsAsPairs.asEvent);
```

Comment:
better remove `asEvent` here. In`functionPerformWith`, which is the latest point of conversion, it can also be a key value pair, because there, `keysValuesDo` is called, which works for both anyway. This also saves some memory for conversions.